Sources/Filters/General/PaintFilter/PaintFilter.worker.js

```diff
@@ -41,35 +41,37 @@ function handlePaintRectangle({ point1, point2 }) {
 
 function handlePaintEllipse({ center, scale3 }) {
   const radius3 = [...scale3];
+  const indexCenter = center.map((val) => Math.round(val));
+
   if (globals.slicingMode != null && globals.slicingMode !== SlicingMode.NONE) {
     const sliceAxis = globals.slicingMode % 3;
-    radius3[sliceAxis] = 0.25;
+    radius3[sliceAxis] = 1;
   }
 
   const yStride = globals.dimensions[0];
   const zStride = globals.dimensions[0] * globals.dimensions[1];
 
-  const zmin = Math.round(Math.max(center[2] - radius3[2], 0));
+  const zmin = Math.round(Math.max(indexCenter[2] - radius3[2], 0));
```

Comment:
if `center = [0, 0, 10]` and `spacing = [0,0, 0.4]`, then slice 9 should get painted (at least a dot)